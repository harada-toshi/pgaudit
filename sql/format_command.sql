\! cp sql/format_command/format_command.conf /tmp/config.conf
\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log
\! pg_ctl -w restart -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! sh sql/format_command/format_command.sh
