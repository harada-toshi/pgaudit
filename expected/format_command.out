	        format = 'AUDIT: %class,%command_tag,%command_parameter,%command_text'
LOG:  AUDIT: MISC,BEGIN,,BEGIN;
LOG:  AUDIT: READ,PREPARE,,PREPARE prep_dx AS SELECT * FROM pg_extension WHERE extname = $1 AND extversion = $2;
LOG:  AUDIT: READ,SELECT,plpgsql 1.0,PREPARE prep_dx AS SELECT * FROM pg_extension WHERE extname = $1 AND extversion = $2;
LOG:  AUDIT: MISC,EXECUTE,,EXECUTE prep_dx ('plpgsql','1.0');
LOG:  AUDIT: MISC,DEALLOCATE,,DEALLOCATE PREPARE prep_dx;
LOG:  AUDIT: MISC,COMMIT,,END;
